swimlane/core/resources/report.py

```diff
@@ -71,12 +71,17 @@ class Report(APIResource, PaginatedCursor):
     )
 
     default_limit = 50
+    pstart = 0
```

Comment:
Adhere to naming convention and default value as mentioned in previous comments.